content/src/view.js

```diff
@@ -3172,6 +3380,147 @@ function setupHpanelBox(box) {
   });
 }
 
+function curvedVertical(x1, y1, x2, y2) {
+  var radius = Math.abs(y1 - y2);
+  var line = [];
+
+  return 'M'+ x1 + "," + y1 + " " + 'A'+ radius + "," + radius + " 1 0,1 " + x2 + "," + y2;
+}
+
+//$(window).resize(..) event
+
+function arrow(pane, arrows, traceEventNum, show_fade){
+  /* note: we expect arrow_lines to be an array of key value pairs where 
+  each key is a color for the arrow, and each value is a list of location pairs
+  to draw an arrow on.   */
+
+  var arrow_data = arrows[traceEventNum]; 
```

Comment:
Very neat.  Can you do me a favor and add a big comment here illustrating the meaning of each of these variables?  For example, you could give example instance values for the whole set of variables, and  then illustrate with ASCII art what is going on (e.g., with x y coordinates) for that set of instance variables.